onnx_xla/backend.cc

```diff
@@ -257,7 +249,7 @@ namespace onnx_xla {
         }   
         auto l_ptr = executor_->descriptorToLiteral(t);
         auto constant = builder_.ConstantLiteral(*l_ptr);
```

Comment:
What's the type of constant?

Reply:
Hmm, I don't see `XlaOp` has rvalue operator= defined. It really just have two pointers inside. I think you can do without `std::move`.

Reply:
I suppose compiler has been creating a default rvalue assignment, but its a small object, so changed